rllib/examples/envs/classes/multi_agent/tic_tac_toe.py

```diff
@@ -153,8 +161,8 @@ def render(self) -> str:
 
         Returns:
             A string representation of the board where:
-            - 'X' represents player1's pieces
-            - 'O' represents player2's pieces
+            - 'X' represents the current player's pieces
+            - 'O' represents opponent player's pieces
```

Comment:
this is much cleaner description

Reply:
Previously the agents couldn't learn because it didn't know if it was noughts or the crosses